blurb/blurb.py

```diff
@@ -677,6 +677,8 @@ def help(subcommand=None):
         summaries = []
         longest_name_len = -1
         for name, fn in subcommands.items():
+            if name == '--help':
```

Comment:
Can you change this to `if name.startswith("-"):` ?  That would let us support `--version`` too.

Reply:
Done. (There isn't currently a "version" subcommand though.)